README.md

```diff
@@ -1,13 +1,12 @@
 # Project Name
 
-Replace this readme with your own information about your project.
+This is a version of Guess Who game. The user must identify a person based on three guesses. Then the user must make a choice. We are using javascript conditionals , arrys and objects. 
 
```

Comment:
I think I can make more than three guesses. But that was not in the scope anyhow!